src/com/jwetherell/algorithms/mathematics/Division.java

```diff
@@ -22,6 +28,9 @@ public static final long divisionUsingLoop(int a, int b) {
     }
 
     public static final long divisionUsingRecursion(int a, int b) {
+        if (b == 0) {
+            throw new IllegalArgumentException("Cannot divide by zero");
+        }
```

Comment:
Checks if b is equal to 0, then if true, throw an `IllegalArgumentException` in divisionUsingRecursion function